serverless/messaging/reference-content/sns-overview.mdx

```diff
@@ -0,0 +1,57 @@
+---
+meta:
+  title: SNS Overview
+  description: Learn about the SNS messaging systems
+content:
+  h1: SNS Overview
+  paragraph: Learn about the SNS messaging systems
+tags: messaging sns aws queuing publish-subscribe message-queue topic notification simple notification service
+categories:
+  - serverless
+dates:
+  validation: 2023-10-05
+  posted: 2023-01-04
+---
+
+## What is SNS?
+
+SNS is a managed messaging service developed by AWS and supported by Scaleway Messaging and Queuing. It is often coupled to work with [SNS](/serverless/messaging/reference-content/sns-overview/).
```

Comment:
Does "It" refer to SNS or Messaging and Queuing? Or should it be "It is often coupled to work with SQS" instead?